docs/guides/04-data-driven-visualizations-part-1.md

```diff
@@ -46,7 +46,7 @@ color: ramp($population_density, [midnightblue, deeppink, gold])
 ```
 #### Implicit casts
 
-Matching the input with the context of the lowest population density and highest population density is done by the [`linear`](/developers/carto-vl/reference/#cartoexpressionslinear) function, which is used automatically by `ramp` when the input is a numeric property. This means that the CARTO VL `ramp` function makes transformations that we call *implicit casts*.
+Matching the input with the context of the lowest population density and highest population density is done by the [`linear`](/developers/carto-vl/reference/#cartoexpressionslinear) function, which is used automatically by `ramp` when the input is a numeric property. This means that the CARTO VL `ramp` function makes transformations that we call [implicit casts](developers/carto-vl/guides/Glossary/#implicit-cast).
```

Comment:
It looks like we're missing the initial "/" for implicit casts

Reply:
That detail and then we're ready to merge!

Reply:
oops! ok, fixing real quick!